src/controllers/csi/metadata/sqlite_test.go

```diff
@@ -118,20 +118,69 @@ func TestUpdateDynakube(t *testing.T) {
 	assert.Equal(t, name, testDynakube1.Name)
 }
 
-func TestGetDynakubes(t *testing.T) {
+func TestGetTenantsToDynakubes(t *testing.T) {
 	db := FakeMemoryDB()
 	err := db.InsertDynakube(&testDynakube1)
 	assert.Nil(t, err)
 	err = db.InsertDynakube(&testDynakube2)
 	assert.Nil(t, err)
 
-	dynakubes, err := db.GetDynakubes()
+	dynakubes, err := db.GetTenantsToDynakubes()
 	assert.NoError(t, err)
 	assert.Equal(t, 2, len(dynakubes))
 	assert.Equal(t, testDynakube1.TenantUUID, dynakubes[testDynakube1.Name])
 	assert.Equal(t, testDynakube2.TenantUUID, dynakubes[testDynakube2.Name])
 }
 
+func TestGetAllDynakubes(t *testing.T) {
+	db := FakeMemoryDB()
+	err := db.InsertDynakube(&testDynakube1)
+	assert.Nil(t, err)
+	err = db.InsertDynakube(&testDynakube2)
+	assert.Nil(t, err)
+
+	dynakubes, err := db.GetAllDynakubes()
+	assert.NoError(t, err)
+	assert.Equal(t, 2, len(dynakubes))
```

Comment:
Would even check if the returned objects are dk1 & dk2

Similar for the next tests